controllers/profiles/following.js

```diff
@@ -8,14 +8,35 @@ const {QueryTypes} = require('sequelize');
 const config = require('../../databases/config/database')[env];
 
 module.exports = async (req, res) => {
-  const id = req.userId;
-  console.log(id);
+  const userId = req.userId;
+
   let sequelize;
   if (config.use_env_variable) {
     sequelize = new Sequelize(process.env[config.use_env_variable], config);
   } else {
     sequelize = new Sequelize(config.database, config.username, config.password, config);
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The creation of a new Sequelize instance inside the route handler is not efficient. This operation should be done once during the application startup and reused across different parts of the application. Consider moving this to a separate module and import it where needed.
```diff
-   let sequelize;
-   if (config.use_env_variable) {
-     sequelize = new Sequelize(process.env[config.use_env_variable], config);
-   } else {
-     sequelize = new Sequelize(config.database, config.username, config.password, config);
-   }
+   const sequelize = require('../../databases/sequelize');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->